osu.Game/Overlays/OverlayPanelDisplayStyleControl.cs

```diff
@@ -78,7 +78,7 @@ public PanelDisplayTabItem(OverlayPanelDisplayStyle value)
                         RelativeSizeAxes = Axes.Both,
                         FillMode = FillMode.Fit
                     },
-                    new HoverClickSounds()
+                    new HoverSounds()
```

Comment:
Why isn't the `TabSelect` style? And why can't it be shared in `OsuTabControl`.

Reply:
That's how it was previously.